src/jsx/components/input_info.jsx

```diff
@@ -105,9 +105,13 @@ class InputInfo extends React.Component {
                   </a>
                 </li>
               ) : null}
-              <li className="dropdown-item">
-                <a onClick={() => this.saveTheJSON()}>Save JSON</a>
-              </li>
+              {this.props.platform != "gui"
```

Comment:
Why isn't saving the JSON allowed on gui?

Reply:
I modified it to allow always allow saving the JSON, including on the gui. Good catch.